### ISSUE TITLE
A11y - Red error text contrast issue in dark mode

### ISSUE THREAD

**Issue by showeimer** [IBM/kui#3096]
Accessibility Violations

Error response text has a contrast of 4.38, requirement is 4.5.

![red](https://user-images.githubusercontent.com/26075187/67504019-a9ecce80-f656-11e9-8e29-81467eafb935.png)

@cwyckoff3 previous issue mistakenly got closed due to the second problem being fixed; opening this issue so that this is still tracked.

**Comment by mattcardinal** [comment]
@alexandermelox

**Comment by mattcardinal** [comment]
The decision of our visual designers is to replace red 50 with red 40 #FF767C This red is not in the carbon color library, but is in the IBM design language color library.

I will bring it up with the carbon team and see if we can get an issue open for it in their repo.
@cwyckoff3

**Comment by myan9** [comment]
@mattcardinal, thanks! 

I pushed a PR replacing `red 50` with `red 40`. I think maybe it's worth changing `green 50` as well. In the table below, `red 40` doesn't seem to match `green 50`, and the contrast between them is lower than before, which makes it a little difficult for users to differentiate the rows and mentally parse the table.

![image](https://user-images.githubusercontent.com/21212160/68251101-7021ad80-fff0-11e9-867a-2d6117118981.png)
